ktor-client/ktor-client-tests/common/src/io/ktor/client/tests/utils/CommonClientTestUtils.kt

```diff
@@ -109,8 +109,8 @@ private suspend fun concurrency(level: Int, block: suspend (Int) -> Unit) {
     }
 }
 
-class TestClientBuilder<T : HttpClientEngineConfig>(
-    var config: HttpClientConfig<T>.() -> Unit = {},
+class TestClientBuilder<out T : HttpClientEngineConfig>(
+    var config: HttpClientConfig<@UnsafeVariance T>.() -> Unit = {},
```

Comment:
The alternative would be to change `ClientLoader.clientTests` to use `TestClientBuilder<out HttpClientEngineConfig>` but that looks more like a breaking API change. This solution is more general.

Should I even switch `HttpClientConfig` to use `out T` instead? There are a few places in the code that use explicit `HttpClientConfig<out T>` which would become redundant, but also a few more places that would require `@UnsafeVariance`. Not much, but maybe there was a reason why HttpClientConfig wasn't `out T` in the first place? I don't know the project too well. :)